config.yaml

```diff
@@ -6,14 +6,22 @@ options:
     type: boolean
     default: true
     description: Allows dex to keep a list of passwords which can be used to login to dex
+  issuer-url:
```

Comment:
This is a good suggestion, but it is a pattern we are haven't followed in the past. Do you have an example of this?

Reply:
We can simply leave the config option (add deprecation warning to option description) there and use something like this.

```python
    @property
    def _issuer_url(self) -> str:
        """Return issuer-url value if config option exists; otherwise default Dex's endpoint."""
        if issuer_url := self.model.config["issuer-url"]:
            return issuer_url

        if public_url := self.model.config["public-url"]:
            self.logger.warning(
                "DEPRECATED: The public-url config option is depracted, please use issuer-url."
            )
            if not public_url.startswith(("http://", "https://")):
                public_url = f"http://{public_url}"

            return f"{public_url}/dex"

        return (
            f"http://{self.model.app.name}.{self._namespace}.svc:{self.model.config['port']}/dex"
        )
```

Reply:
I think this is a good suggestion, but I will work on it in a separate PR to avoid pushing commits directly into the dev branch.

Reply:
@rgildein Let's track this change in https://github.com/canonical/dex-auth-operator/pull/213

Reply:
Resolving as the PR got merged.